src/services/TechRecordsService.ts

```diff
@@ -88,6 +88,12 @@ class TechRecordsService {
     delete techRecordItem.secondaryVrms; // No longer needed
     delete techRecordItem.partialVin; // No longer needed
 
+    techRecordItem.techRecord.forEach((techRecord) => {
+      if (techRecord.euroStandard !== undefined && techRecord.euroStandard !== null) {
```

Comment:
I did not use the Truthy check here because 0 is a possible value and 0 is Falsy